test-electron-supabase.js

```
@@ -0,0 +1,277 @@
+    const { data, error } = await window.supabaseService.insert('users', testData);
+    
+    if (error) {
+      console.log('⚠️  Sync to Supabase failed:', error.message);
+      return false;
+    } else {
+      console.log('✅ Sync to Supabase successful');
+      
+      // Clean up - delete the test record
+      if (data && data[0]) {
+        await window.supabaseService.delete('users', data[0].id);
+        console.log('✅ Test record cleaned up');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)